addons/mail/static/src/core/common/message.xml

```diff
@@ -154,27 +154,16 @@
 </t>
 
 <t t-name="mail.Message.actions">
-    <div t-if="props.hasActions and message.hasActions and !isEditing and !env.inChatter?.disabled" class="o-mail-Message-actions d-print-none d-flex align-items-start"
+    <div t-if="props.hasActions and message.hasActions and !isEditing and !env.inChatter?.disabled" class="o-mail-Message-actions d-print-none d-flex align-items-start mx-1"
         t-att-class="{
             'start-0': isAlignedRight,
-            'mx-1': !isMobileOS,
             'mt-1': message.bubbleColor and !props.asCard and !isMobileOS,
             'my-n2': !message.bubbleColor and !props.asCard and !isMobileOS,
             'invisible': !isActive and !isMobileOS,
             'o-expanded': optionsDropdown.isOpen
         }"
     >
-        <t t-if="isMobileOS">
-            <t t-if="isAlignedRight">
-                <t t-call="mail.Message.emptyQuickAction"/>
-                <t t-if="!props.asCard" t-call="mail.Message.expandAction"/>
-            </t>
-            <t t-else="">
-                <t t-call="mail.Message.expandAction"/>
-                <t t-if="!props.asCard" t-call="mail.Message.emptyQuickAction"/>
-            </t>
-        </t>
-        <t t-else="">
```

Comment:
Empty quick actions must be kept or the message bubbles go in the enemy territory:
![IMG_388E16F6DA51-1](https://github.com/user-attachments/assets/3d85adb8-09f9-488c-a8f4-27199411a675)